src/QsCompiler/CompilationManager/AssemblyLoader.cs

```diff
@@ -27,15 +27,20 @@ namespace Microsoft.Quantum.QsCompiler
     public static class AssemblyLoader
     {
         /// <summary>
-        /// Loads the Q# data structures in a referenced assembly given the Uri to that assembly,
-        /// and returns the loaded content as out parameter.
-        /// Returns false if some of the content could not be loaded successfully,
+        /// Loads the Q# data structures in a referenced assembly given <paramref name="asm" />,
+        /// and returns the loaded content as <paramref name="headers" />.
+        /// </summary>
+        /// <param name="asm">The uri of the referenced assembly.</param>
+        /// <returns>
+        /// False if some of the content could not be loaded successfully,
         /// possibly because the referenced assembly has been compiled with an older compiler version.
-        /// If onDeserializationException is specified, invokes the given action on any exception thrown during deserialization.
+        /// </returns>
+        /// <exception cref="FileNotFoundException"><paramref name="asm" /> does not exist.</exception>
+        /// <exception cref="ArgumentException"><paramref name="asm" /> is not an absolute file URI.</exception>
+        /// <remarks>
+        /// If <paramref name="onDeserializationException" /> is specified, it will be invoked on any exception thrown during deserialization.
```

Comment:
Maybe this sentence could be the param tag for `onDeserializationException`? I saw you mentioned that you didn't add param tags for all methods, I'm not sure if that applies to methods that would need new documentation to be written for the params, or also ones with existing text that could be repurposed. Either way is fine. :)

Reply:
This is definitely one that I missed :) In general, I opted to favor repurposing any text into parameter tags where possible, but I wasn't as strict about doing that in the beginning. I'll add a comment below (hopefully tomorrow) with some of the decisions I found myself making :D